.github/workflows/scheduled-pr-reminders.yml

```diff
@@ -0,0 +1,30 @@
+name: scheduled-pr-reminders
+
+permissions: read-all
+
+on:
+  schedule:
+    - cron: '37 9 * * *'
```

Comment:
Out of curiosity, why is this particular time chosen?

Reply:
The time is in UTC; after 9am should mean that the action always runs on the same day in PDT as UTC (in case that could end up being confusing) while still running early in the day for folks in the UK. `37` is arbitrary just to avoid running on the hour in case that's crowded.

Reply:
yea, makes sense. Thanks